app/src/main/res/values/dimens.xml

```diff
@@ -24,6 +24,15 @@
   <dimen name="grid_35">280dp</dimen>
   <dimen name="grid_40">320dp</dimen>
 
+  <!-- Kickstarter six-point grid keylines -->
```

Comment:
ehh, something that would be nice to figure out maybe before merging this is if hot swapping out our legacy 8pt grid system with 6pt values would mess everything up....maybe i'll experiment with that while this is being reviewed 🤔